Quizzler-iOS13/SceneDelegate.swift

```diff
@@ -13,11 +13,34 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     var window: UIWindow?
 
 
-    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+    func scene(
+        _ scene: UIScene,
+        willConnectTo session: UISceneSession,
+        options connectionOptions: UIScene.ConnectionOptions
+    ) {
         // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
         // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
         // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
-        guard let _ = (scene as? UIWindowScene) else { return }
+        
+        
+        // 1. Capture the scene if there is one into a variable
+        guard let windowScene = (scene as? UIWindowScene) else { return }
+        
+        // 2. Create a new UIWindow using the windowScene constructor which takes in a window scene
+        let window = UIWindow(windowScene: windowScene)
+        
+        // 3. Create the view hierarchie programmatically
+        let viewController = ViewController()
```

Comment:
Does it actually work? Given that you have your UI inside Storyboard, you should initialize your ViewController from storyboard. 

`let viewController = UIStoryboard(name: "Main", bundle: nil).instantiateViewController(withIdentifier: "ViewController")`

This might give you a hint:
https://medium.com/@vasanthkumarvelusamy/programmatic-entry-to-viewcontroller-in-storyboard-on-app-launch-51c3281ceea2

Reply:
I updated this on the recent commit but this error is showing. 

"Cannot convert value of type 'UIViewController.Type' to expected argument type 'UIViewController'"